### PR TITLE
feat: Warning prompt

### PR DESCRIPTION
**Pull request by cvaldez98**
I introduced a `confirmationPrompt` element that is a step down from the default `warningPrompt`. It uses a less aggressive prompt for the users which can be used for smaller, less important cases. 
This is shown in verifyRemoteReload which now uses `confirmationPrompt` instead of `warningPrompt` as well as a more descriptive error message.
## Old confirmation prompt style: 
(scary and looks really important)
image

## New confirmation prompt style: 
image

### CODE REVIEW

File: src/renderer/components/dialogs.tsx [hunk 1]
```diff
@@ -31,13 +32,17 @@ export class Dialogs extends React.Component<DialogsProps, {}> {
     const maybeAddLocalVersion = isAddVersionDialogShowing
       ? <AddVersionDialog key='add-version-dialog' appState={appState} />
       : null;
+    const eitherWarningOrPrompt = appState.isWarningDialogShowing
```
Comment (cvaldez98) [inline src/renderer/components/dialogs.tsx]: Open to suggestions here!

File: src/renderer/components/commands-address-bar.tsx [hunk 2]
```diff
@@ -82,8 +82,7 @@ export class AddressBar extends React.Component<AddressBarProps, AddressBarState
   }
 
   public async loadFiddleFromElectronExample(_: any, exampleInfo: { path: string; ref: string }) {
-    console.log(`Loading fiddle from Electron example`, _, exampleInfo);
```
Comment (felixrieseberg) [inline src/renderer/components/commands-address-bar.tsx]: Did you mean to to cut this line?

File: src/renderer/components/commands-address-bar.tsx [hunk 3]
```diff
@@ -101,15 +100,15 @@ export class AddressBar extends React.Component<AddressBarProps, AddressBarState
    *
    * @param what What are we loading from (gist, example, etc.)
    */
-  public async verifyRemoteLoad(what: string): Promise<boolean> {
+  public async verifyRemoteLoad(what: string, fiddlePath?: string): Promise<boolean> {
     const { appState } = this.props;
-    appState.setWarningDialogTexts({
-      label: `Are you sure you sure you want to load this ${what}? Only load and run it if you trust the source`
+    appState.setConfirmationPromptTexts({
+      label: `Are you sure you sure you want to load this \'${what}\' from fiddle path \'${fiddlePath}?\' Only load and run it if you trust the source.`
```
Comment (felixrieseberg) [inline src/renderer/components/commands-address-bar.tsx]:
```suggestion
      label: `Are you sure you sure you want to load this '${what}' from fiddle path '${fiddlePath}'? Only load and run it if you trust the source.`
```

File: src/renderer/components/dialog-confirm.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+export interface ConfirmDialogProps {
+  appState: AppState;
+}
+
+export interface ConfirmDialogState {
+}
+
+/**
+ * The token dialog prompts the user to either continue or cancel the operation.
+ * It's also responsible for checking if the token is correct.
```
Comment (felixrieseberg) [inline src/renderer/components/dialog-confirm.tsx]:
```suggestion
```

File: src/renderer/components/dialog-warning.tsx [hunk 5]
```diff
@@ -13,7 +13,7 @@ export interface WarningDialogState {
 }
 
 /**
- * The token dialog asks the user for a GitHub Personal Access Token.
+ * The token dialog prompts the user to either continue or cancel the operation.
  * It's also responsible for checking if the token is correct.
```
Comment (felixrieseberg) [inline src/renderer/components/dialog-warning.tsx]:
```suggestion
```

File: src/renderer/components/dialog-warning.tsx [hunk 6]
```diff
@@ -35,7 +35,7 @@ export class WarningDialog extends React.Component<WarningDialogProps, WarningDi
 
   public render() {
     const { isWarningDialogShowing, warningDialogTexts } = this.props.appState;
-
+
```
Comment (felixrieseberg) [inline src/renderer/components/dialog-warning.tsx]:
```suggestion

```

**Pull request merged**